.travis.yml

```diff
@@ -28,23 +26,34 @@ jobs:
 
   - name: Backend E2E Tests - Openshift
     script:
+    - travis_retry scripts/setup-tools-for-ubuntu.sh
     - travis_retry scripts/setup-openshift.sh ${OC_TOKEN}
     - make -C enterprise-suite openshift-backend-tests
 
   - name: Backend E2E Go Tests - Minikube
+    before_script:
+    - echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin ${DOCKER_REGISTRY}
```

Comment:
Why not just put the build container in the public bintray repo? Does it really need to be private? I think the build container repo could be public too, like console-charts is.

Reply:
I did make the repo public initially but then got an email from @JustinPihony about that. The policy is to make everything private unless there's a good reason for it to be public.